src/Resources/config/hooks.yml

```diff
@@ -0,0 +1,3 @@
+services:
```

Comment:
Please use the `listeners.xml` and remove the `hooks.yml` again.

Reply:
Removed the file